components/client-attestation-mgt/org.wso2.carbon.identity.client.attestation.mgt/src/main/java/org/wso2/carbon/identity/client/attestation/mgt/model/AppleAttestationMetaData.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.client.attestation.mgt.model;
+
+/**
+ * Represents metadata related to Apple app attestation.
+ */
+public class AppleAttestationMetaData {
```

Comment:
What is the impact of merging the AppleAttestationMetaData and AndroidAttestationMetaData and using a general variable like appIdentifier?

Reply:
We don't these models at all, cause validation requires relevent SP_Properties and attestation header. Having addtional model is not optimum